src/nile/core/account.py

```diff
@@ -3,10 +3,12 @@
 import os
 
 from dotenv import load_dotenv
+from starkware.starknet.compiler.compile import compile_starknet_files
 
 from nile import accounts, deployments
-from nile.common import get_nonce
+from nile.common import CONTRACTS_DIRECTORY, UNIVERSAL_DEPLOYER_ADDRESS, get_nonce
```

Comment:
`get_nonce` is part of `nile.utils` now (because is a command). And we should use get_nonce_without_log (as in the `send` method) to avoid logging nonce from the cli in the declare command.

Regarding where `get_nonce` should be (maybe in the commands folder inside core), this is related to this issue that is assigned to the **after** milestone: #195 .